src/itinerary/Map.svelte

```diff
@@ -0,0 +1,101 @@
+<script>
+  import { voyage } from "./../store/index.js";
+  import panzoom from "panzoom";
+  import { push } from "svelte-spa-router";
+  import { onMount } from "svelte";
+  import rough from "roughjs/bundled/rough.esm.js";
+  import { SVG } from "@svgdotjs/svg.js";
+  export let excursions;
+  const seed = { seed: 1337 };
```

Comment:
As from my commits IRL - I believe we can make some small changes to make the islands more realistic. This can come later as a polishing step however. The core functionality seems good. 

I don't really understand how this code works but IDK if I need to atm.